### PR TITLE
Add cassandra plugin

### PR DESCRIPTION
**Pull request by calvn**

### CODE REVIEW

**Comment by jefferai** [comment]
Hi Calvin, nice to see you back! I've started review of the other pr so please do not merge this in anywhere until the other is merged into master!

**Comment by calvn** [comment]
Glad to be back! No problem, I'll mark you and Brian as reviewers for approval whenever it's good to merge.

File: plugins/database/cassandra/cassandra.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	defaultRollbackCQL = `DROP USER '{{username}}';`
+	cassandraTypeName  = "cassandra"
+)
+
+type Cassandra struct {
+	connutil.ConnectionProducer
+	credsutil.CredentialsProducer
+}
+
+func New() *Cassandra {
```
Comment (briankassouf) [inline plugins/database/cassandra/cassandra.go]: The function definition changed slightly last week, checkout the other plugins but I think this should be `New() (interface{}, error)`

Comment (calvn) [inline plugins/database/cassandra/cassandra.go]: Thanks, noted!

File: plugins/database/cassandra/cassandra.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+	return username, password, nil
+}
+
+func (c *Cassandra) RenewUser(statements dbplugin.Statements, username string, expiration time.Time) error {
+	// NOOP
+	return nil
+}
+
+func (c *Cassandra) RevokeUser(statements dbplugin.Statements, username string) error {
```
Comment (jefferai) [inline plugins/database/cassandra/cassandra.go]: This function totally ignores any revocation statements set by the admin.

**Comment by calvn** [comment]
Do you want those changes applied here, or post-merge?

**Comment by jefferai** [comment]
Either way -- if you want to merge it, can you add a comment to that code post-merge so that we can make sure it gets tracked?

**Pull request merged**